src/modules/direction/directionReducer/actionTypes.ts

```diff
@@ -0,0 +1,24 @@
+export enum LoadData {
+  LOAD_MATERIALS = 'LOAD_MATERIALS',
+  LOAD_EXPERTS = 'LOAD_EXPERTS',
+}
+
+export interface IMaterialsItem {}
+
+export interface IExpertsItem {}
+
+export interface MainState {
+  materials: IMaterialsItem[];
+  experts: IExpertsItem[];
+}
```

Comment:
let's move this one to reducer, since only **action**Types are stored here

Reply:
moved